fire/cli/niv/ilæg_revision.py

```diff
@@ -404,30 +393,98 @@ def ilæg_revision(
             continue
-    opret_tekst = f"Opretter {len(til_opret)} attributter fordelt på {len(punkter_med_oprettelse)} punkter"
-    sluk_tekst = f"Slukker for {len(til_sluk)} attributter fordelt på {len(punkter_med_slukning)} punkter"
-    ret_tekst = f"Retter {len(til_ret)} attributter fordelt på {len(punkter_med_rettelse)} punkter"
 
-    if test:
-        fire.cli.print(
-            f"TESTEDE punktrevision for '{projektnavn}':", fg="yellow", bold=True
+    fikspunktstyper = [FikspunktsType.GI for _ in nye_punkter]
+    landsnumre = fire.cli.firedb.tilknyt_landsnumre(nye_punkter, fikspunktstyper)
+    til_opret.extend(landsnumre)
+    for p in nye_punkter:
+        punkter_med_oprettelse.add(p.ident)
+
+    if len(til_opret) > 0 or len(til_ret) > 0:
+        sagsevent = Sagsevent(
+            id=uuid(),
+            sagsid=sag.id,
+            sagseventinfos=[
+                SagseventInfo(beskrivelse="Opdatering af punktinformationer")
+            ],
+            eventtype=EventType.PUNKTINFO_TILFOEJET,
+            punktinformationer=[*til_opret, *til_ret],
         )
-        if koordinatoprettelsestekst != "":
-            fire.cli.print(f"    * {koordinatoprettelsestekst}")
-        fire.cli.print(f"    * {opret_tekst}")
-        fire.cli.print(f"    * {sluk_tekst}")
-        fire.cli.print(f"    * {ret_tekst}")
-        fire.cli.print(f"Ingen data lagt i FIRE-databasen", fg="yellow")
-        sys.exit(0)
 
+        fire.cli.firedb.indset_sagsevent(sagsevent, commit=False)
+        sagsgang = opdater_sagsgang(sagsgang, sagsevent, sagsbehandler)
+        flush()
 
-def opret_punkt(ident: str, lokation: str, sag: Sag):
-    """Opret nyt punkt i databasen, ud fra minimumsinformationsmængder."""
+    if len(til_sluk) > 0:
+        sagsevent = Sagsevent(
+            id=uuid(),
+            sagsid=sag.id,
+            sagseventinfos=[SagseventInfo(beskrivelse="Lukning af punktinformationer")],
+            eventtype=EventType.PUNKTINFO_FJERNET,
+            punktinformationer_slettede=til_sluk,
+        )
+
+        fire.cli.firedb.indset_sagsevent(sagsevent, commit=False)
+        sagsgang = opdater_sagsgang(sagsgang, sagsevent, sagsbehandler)
+        flush()
+
+    opret_tekst = f"- oprette {len(til_opret)} attributter fordelt på {len(punkter_med_oprettelse)} punkter"
+    sluk_tekst = f"- slukke for {len(til_sluk)} attributter fordelt på {len(punkter_med_slukning)} punkter"
+    ret_tekst = f"- retter {len(til_ret)} attributter fordelt på {len(punkter_med_rettelse)} punkter"
+
+    fire.cli.print("")
+    fire.cli.print("-" * 50)
+    fire.cli.print("Punkter færdigbehandlet, klar til at")
+    fire.cli.print(opret_tekst)
+    fire.cli.print(sluk_tekst)
+    fire.cli.print(ret_tekst)
+
+    spørgsmål = click.style(
+        f"Er du sikker på du vil indsætte ovenstående i {fire.cli.firedb.db}-databasen",
+        fg="white",
+        bg="red",
+    )
+    if bekræft(spørgsmål):
+        fire.cli.firedb.session.rollback()
```

Comment:
Skulle det ikke være session.commit() her?

Reply:
Jo! Godt fanget. Det var ekstra sikkerhed for mig selv indtil jeg fik bekræftelsemekanismen helt på plads.